src/xrpld/rpc/detail/RPCCall.cpp

```diff
@@ -360,7 +416,12 @@ class RPCParser
 
         std::string input = jvParams[0u].asString();
         if (input.find_first_not_of("0123456789") == std::string::npos)
-            jvRequest["can_delete"] = jvParams[0u].asUInt();
+        {
+            if (auto seq = jvParseUInt(jvParams[0u]))
```

Comment:
And I missed this too:

```suggestion
            if (auto const seq = jvParseUInt(jvParams[0u]))
```